sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -391,14 +395,16 @@ multilib_src_install_all() {
 	keepdir /var/lib/systemd
 	keepdir /var/log/journal
 
+	if use kernel-install; then
+		keepdir usr/lib/kernel/install.d
+	fi
+
 	if use pam; then
 		newpamd "${FILESDIR}"/systemd-user.pam systemd-user
 	fi
 
-	if use boot; then
-		python_fix_shebang "${ED}"
-		secureboot_auto_sign
-	fi
+	use ukify && python_fix_shebang "${ED}"
```

Comment:
I had this same thought earlier, but then I got confused when I noticed that systemd-utils also depends on the same python stuff, even though it doesn't install ukify. This might simply be a mistake though, since as far as I can see neither systemd-boot nor kernel-install does anything with python at all.

Reply:
Ah, could definitely be a mistake on my part.

Reply:
Fixed!